cadquery/occ_impl/geom.py

```diff
@@ -619,19 +619,28 @@ def rotated(self, rotate=(0, 0, 0)):
         :param rotate: Vector [xDegrees, yDegrees, zDegrees]
         :return: a copy of this plane rotated as requested.
         """
-        rotate = Vector(self.toWorldCoords(rotate))
+        rotate = Vector(rotate)
```

Comment:
Could you add a comment here:
`#NB: this is not a geometric Vector`

Reply:
b78d4f2b4cf6ccb86a2fae7ddd6e5ee552effa5a